app/controllers/sign_up/completions_controller.rb

```diff
@@ -55,8 +59,9 @@ def loa3?
       sp_session[:loa3] == true
     end
 
-    def service_provider_attributes
-      { loa3: sp_session[:loa3], service_provider_name: decorated_session.sp_name }
+    def return_to_account
+      track_completion_event('account')
```

Comment:
Looks like this one might be double tracked: https://github.com/18F/identity-idp/pull/2905/files#diff-cd7bc1d152ba976bc3f9dac0281ca675R18